tests/strip_notebook_outputs.py

```diff
@@ -0,0 +1,58 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+# Script to strip outputs of Jupyter notebooks except for plots and prints so that there
+# are displayed nicely on the documentation website.
+
+import os
+
+import nbformat
+from nbconvert.preprocessors import Preprocessor
+
+
+class StripOutputExceptPlotsAndPrintsPreprocessor(Preprocessor):
```

Comment:
please add some (even rudimentary) docstring

Reply:
Also in the methods below

Reply:
will do.